docs/getting-started/run-applications.md

```diff
@@ -33,12 +33,38 @@ on-demand by Crossplane.
 > published a PostgreSQLInstance with at least one working Composition in order
 > to create the OAM application we'll use in this guide.
 
-## Install the OAM Addon
+## Infrastructure Operator
+
+### Install OAM build-in workloads and traits
 
 As the infrastructure operator our work is almost done - we defined, published,
 and composed the infrastructure  that our application developer and operator
-team-mates will use in the previous guide. One task remains for us, which is to
-install and configure Crossplane's OAM addon.
+team-mates will use in the previous guide. One task remains is, we must define 
+the [_workloads_] and [_traits_] that our platform supports. We'll discuss workloads 
+and traits in more detail later.
+
+The infrastructure operator could choose which workloads
+and traits their platform supports by creating or deleting `WorkloadDefinitions`
+and `TraitDefinitions` like below:
```

Comment:
Is the goal of your proposed sentence to replace the below paragraph, which you've removed further down? Can you explain what your thinking is here?

> Once the addon is installed we must define the [_workloads_] and [_traits_] that our platform supports. OAM applications consist of workloads, each of which may be modified by traits. The infrastructure operator may choose which workloads and traits their platform supports by creating or deleting `WorkloadDefinitions` and `TraitDefinitions`. We'll discuss workloads and traits in more detail below.

Reply:
No. That paragraph is just split and place to different place.

Reply:
Can you help me understand where and why?

As meta feedback it's a little hard to review when the context of this PR is "make the guide better", without explaining what you felt needed improving about the guide or how you feel you've improved it. What is or is not good documentation is highly subjective. I'd definitely open to improving this guide, but some of these changes feel a little arbitrary without context.

Reply:
Can we migrate this comment to https://github.com/crossplane/crossplane/pull/1549#discussion_r427740956?

These two comments are duplicated essentially.